### PR TITLE
The high-level function for group uploading, aka groupFrom

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

**Comment by bautrukevich** [comment]
[UploadClient-a4: The high-level function for group uploading, aka filesGroupFrom](https://trello.com/c/FwpPBv8b/481-uploadclient-a4-the-high-level-function-for-group-uploading-aka-filesgroupfrom)

File: test/checkFileIsUploadedFromUrl.spec.ts [hunk 1]
```diff
@@ -2,23 +2,31 @@ import * as factory from './_fixtureFactory'
 import checkFileIsUploadedFromUrl from '../src/checkFileIsUploadedFromUrl'
 import {StatusEnum} from '../src/api/fromUrlStatus'
 import {getSettingsForTesting} from './_helpers'
+import fromUrl from '../src/api/fromUrl'
 
-describe('checkFileIsUploadedFromUrl', () => {
+describe('checkFileIsUploadedFromUrl', async() => {
```
Comment (nd0ut) [inline test/checkFileIsUploadedFromUrl.spec.ts]: Seems that this function has become `async` by mistake. I haven't found `await` inside and maybe jasmine doesn't work well with async describes. (I'm not sure about that)

File: mock-server/controllers/group.ts [hunk 2]
```diff
@@ -5,12 +5,53 @@ import error from '../utils/error'
 const UUID_REGEX = '[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}'
 const GROUP_ID_REGEX = `${UUID_REGEX}~[1-9][0-9]*$`
```
Comment (nd0ut) [inline mock-server/controllers/group.ts]: Can there be more than 99 files in a group?

File: test/checkFileIsUploadedFromUrl.spec.ts (hunk 1 above)
Comment (bautrukevich) [reply test/checkFileIsUploadedFromUrl.spec.ts]: Yes, you are right! Thank you

**Pull request merged**